src/main/java/frc/robot/subsystems/DriveBase.java

```diff
@@ -68,6 +83,29 @@ private DriveBase() {
 
     }
 
+    public static double getMotorCurrent() {
+        motorCurrent = DriveBase.leftFrontTalon.getSupplyCurrent();
+
+        SmartDashboard.putNumber("MotorCurrent (FrontLeft)", motorCurrent);
+        
+        return(motorCurrent);
+    }
+
+    public static boolean isMotorLimitActive() {
+        //Check if motor current limiting is active (is current draw over or at current limit)
+        if (motorCurrent > Config.kContinCurrentAmps - 1) {
```

Comment:
Is -1 here a tolerance? Is there a reason not to use simply `motorCurrent > Config.kContinCurrentAmps`?

Reply:
@georgetzavelas Yes, the '-1' is a tolerance, if I remember correctly what "tolerance" means. When the motor controller limits current draw, the current draw fluctuates, so I subtract 1 from the limit for a safety margin.

Should I place a comment in the code explaining what the '-1' is for?

Reply:
If it is a tolerance then it would be best to define it as a constant so that the name of the constant makes it clear what its being used for 😄

Reply:
OK. Fixed this, although I might remove it in my next commit (if testing shows that I don't need it).